packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -409,8 +423,16 @@ class TxForm extends Component {
       };
     } else if (balance && balance.lt(amountBn)) {
       return { amount: `You don't have enough ${token.symbol} balance` };
-    } else if (!values.to || !isAddress(values.to)) {
+    } else if (
+      (!values.to || !isAddress(values.to)) &&
+      !isEtcChainId(currentChainIdBN)
```

Comment:
Why do we check this?

Reply:
Sorry it was confusing code.... it was actually just so if the user is connected to the `classic` chain id and they enter an invalid Ethereum address, then it would show a tooltip that reminds them that they need to enter a valid Ethereum address, but it takes the opportunity to highlight to them that the recipient address should be an Ethereum Classic address so they don't lose their funds by sending it to a wrong address.

I've changed it in this commit to make it clearer: https://github.com/paritytech/fether/pull/483/commits/b5296af7259c12f56ca294ef7193d8fbffc5c1b4

Perhaps a future PR could include an upfront warning on that page to remind the user to double check that the recipient address that they enter is on the chain as the chain that they're currently using, and potentially even check that for them automatically.